docs/server/source/http-client-server-api.rst

```diff
@@ -147,7 +147,16 @@ Transactions
    .. literalinclude:: http-samples/post-tx-response.http
       :language: http
 
+   .. note::
+       If the server is returning a ``202`` HTTP status code, then the
+       transaction has been accepted for processing. To check the status of the
+       transaction, poll the link to the `status monitor
+       <https://docs.bigchaindb.com/projects/server/en/latest/http-client-server-api.html#get--api-v1-statuses?tx_id=tx_id>`_
```

Comment:
I will push a commit to fix this hard-coded URL. It's probably not a good idea to use the title of that section as its reference, so I'll create a proper proper reference label instead.